src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/AddPermissionsAndEmployerController.cs

```diff
@@ -64,14 +61,70 @@ public IActionResult Index([FromRoute] int ukprn, AddPermissionsAndEmployerViewM
 
         _sessionService.Set(sessionModel);
 
-        return RedirectToRoute(RouteNames.AddPermissionsAndEmployer, new { ukprn });
+        return RedirectToRoute(RouteNames.AddEmployerAndPermissionsSent, new { ukprn });
     }
 
+    [HttpGet]
+    [Route("/{ukprn}/addEmployer/addPermissions/sent", Name = RouteNames.AddEmployerAndPermissionsSent)]
```

Comment:
Rename the route to AddEmployerConfirmation

Reply:
ok